### DISCUSSION TITLE
UI & UX References

### DISCUSSION THREAD

**Discussion by heyhippari** [jellyfin/jellyfin-meta#3] [General]
The goal of this issue is to establish a list of user interfaces from streaming services, media servers, and media players that we can use as references when designing UI elements and screens around the web client.

The goal is never to make a 1:1 copy of existing services, but look at what is done on these services around the world and use these references to inform decisions.

When submitting screenshots, please be thorough and keep it to one service per post. Make sure to remove any personal information from your screenshots.

**Comment by heyhippari** [comment]
## Shadowz

Shadowz is a small-scale French streaming service specializing in Horror movies.

Platforms available are:
* Desktop
* Chromecast

<details>
<summary>Home</summary>

![Screenshot_2020-04-28 Les meilleurs films d’horreur à voir Shadowz](https://user-images.githubusercontent.com/19396809/80466378-c424a380-893c-11ea-831e-f4c3f1dd6729.jpg)

</details>
<details>
<summary>Category</summary>

The list lazy loads as the user scroll down

![Screenshot_2020-04-28 Films de genre en VOD Horreur, Thriller, Fantastique, Science-fiction Shadowz](https://user-images.githubusercontent.com/19396809/80466804-4e6d0780-893d-11ea-992e-21fc70d142ee.jpg)

</details>
<details>
<summary>Details</summary>

![Screenshot_2020-04-28 Tales from the Crypt](https://user-images.githubusercontent.com/19396809/80466855-5dec5080-893d-11ea-8563-3db3f1728601.png)

</details>
<details>
<summary>Player</summary>

The player has Quality, Audio, Subtitle, and Speed selection, as well as a direct link to subtitle styling options.

![image](https://user-images.githubusercontent.com/19396809/80466953-7e1c0f80-893d-11ea-9a28-75238028ff5b.png)

</details>

**Comment by cromefire** [comment]
Netflix

<details>
<summary>
Desktop:
</summary>

Home:
![Screenshot_2020-04-28 Netflix](https://user-images.githubusercontent.com/26320625/80469807-634b9a00-8941-11ea-84aa-2d4c64f1de9d.png)

Search:
![image](https://user-images.githubusercontent.com/26320625/80470950-ffc26c00-8942-11ea-800a-d32e3f079ca7.png)

Player:
![image](https://user-images.githubusercontent.com/26320625/80470815-cdb10a00-8942-11ea-9571-1d5fbb914614.png)

</details>

<details>
<summary>
Android
</summary>

Home:
![Screenshot_20200428-111601](https://user-images.githubusercontent.com/26320625/80470357-2df37c00-8942-11ea-8021-048798138b95.jpg)

Search:
![Screenshot_20200428-112509](https://user-images.githubusercontent.com/26320625/80471076-30a2a100-8943-11ea-82d5-724716a8de82.jpg)

Player:
![Screenshot_20200428-112609](https://user-images.githubusercontent.com/26320625/80471097-37311880-8943-11ea-8cd8-0e2c273c4793.jpg)

</details>

**Comment by cromefire** [comment]
Disney+

<details>
<summary>
Desktop
</summary>

Home:
https://drive.google.com/open?id=1u6tysYlk4T2cpLd52TDHLPTnzGqBqG8m

Search:
![image](https://user-images.githubusercontent.com/26320625/80476099-35b71e80-894a-11ea-9395-8a7371e852e4.png)

Details:
![image](https://user-images.githubusercontent.com/26320625/80476251-67c88080-894a-11ea-94d7-c74c5d2d2ed3.png)

Player:
![image](https://user-images.githubusercontent.com/26320625/80476286-79aa2380-894a-11ea-801c-44159cb8bd06.png)

</details>

<details>
<summary>
Android
</summary>

Home:
![Screenshot_20200428-113055](https://user-images.githubusercontent.com/26320625/80472473-0e118780-8945-11ea-9c71-77602f5a39ee.jpg)

Search:
![Screenshot_20200428-113121](https://user-images.githubusercontent.com/26320625/80472517-1c5fa380-8945-11ea-9165-62a78acb9273.jpg)

Details:
![Screenshot_20200428-113218](https://user-images.githubusercontent.com/26320625/80472542-2681a200-8945-11ea-8c24-79dd7558069c.jpg)

Player:

N/A

</details>

**Comment by cromefire** [comment]
Amazon Prime

<details>
<summary>
Desktop
</summary>

Home:
![prime-home](https://user-images.githubusercontent.com/26320625/80506632-28b02480-8976-11ea-882b-a81f0f9b580c.png)

Search:
![prime-search](https://user-images.githubusercontent.com/26320625/80506655-306fc900-8976-11ea-95df-01a2ecb203f7.png)

Details:
![prime-details](https://user-images.githubusercontent.com/26320625/80506675-35cd1380-8976-11ea-84c2-cd45ec80c2df.png)

Player:
![prime-player](https://user-images.githubusercontent.com/26320625/80506692-3c5b8b00-8976-11ea-8e52-b50dd6ade049.png)

</details>

<details>
<summary>
Android
</summary>

Home:
![Screenshot_20200428-113334](https://user-images.githubusercontent.com/26320625/80472721-66e12000-8945-11ea-8fc4-f42696385130.jpg)

Search:
![Screenshot_20200428-113407](https://user-images.githubusercontent.com/26320625/80472793-7f513a80-8945-11ea-87dc-9727d594b4f2.jpg)
![Screenshot_20200428-113424](https://user-images.githubusercontent.com/26320625/80472804-837d5800-8945-11ea-8fbb-1ae81f2fd90f.jpg)

Details:
![Screenshot_20200428-113537](https://user-images.githubusercontent.com/26320625/80472815-88420c00-8945-11ea-819b-a852c103f387.jpg)

Player:

N/A

</details>

**Comment by hamburglar2160** [comment]
<details>
<summary>
Amazon Prime (Chromecast Addendum via Android)
</summary>

![image](https://user-images.githubusercontent.com/61879508/80503595-15826200-8940-11ea-81db-62d8ec391565.png)

From the three dotted menu
![image](https://user-images.githubusercontent.com/61879508/80503618-1f0bca00-8940-11ea-9127-3526571d99fa.png)

Cast menu
![image](https://user-images.githubusercontent.com/61879508/80503659-2b902280-8940-11ea-9e7a-5e3dd73df726.png)

</details>

<details>
<summary>
Amazon Prime tv season details (web)
</summary>

![image](https://user-images.githubusercontent.com/61879508/83435931-02fdbb80-a40b-11ea-8db9-592354c07235.png)

</details>








<details>
<summary>
Amazon Prime tv season details (Android)
</summary>

![image](https://user-images.githubusercontent.com/61879508/83436839-98e61600-a40c-11ea-8419-928ccd5a6506.png)

![image](https://user-images.githubusercontent.com/61879508/83436849-9f748d80-a40c-11ea-9089-fcc258f114b4.png)


</details>

**Comment by hamburglar2160** [comment]
<details>
<summary>
HBO
</summary>
<details>
<summary>
Android
</summary>

Home Screen

![image](https://user-images.githubusercontent.com/61879508/80501076-00580400-893d-11ea-9128-242767d171c6.png)

Series Detail
![image](https://user-images.githubusercontent.com/61879508/80501257-3b5a3780-893d-11ea-874c-c882673dce1c.png)
![image](https://user-images.githubusercontent.com/61879508/80501283-42814580-893d-11ea-8b8c-d9a2660929dd.png)

Episode Detail
![image](https://user-images.githubusercontent.com/61879508/80501313-4b721700-893d-11ea-80da-59d74c1dce86.png)

Chromecast Controller
![image](https://user-images.githubusercontent.com/61879508/80501349-54fb7f00-893d-11ea-9d0b-c46ff9af748f.png)
</details>
<details>
<summary>
Web
</summary>

![image](https://user-images.githubusercontent.com/61879508/80504481-2da6b100-8941-11ea-83fc-86bb8b9396e7.png)

![image](https://user-images.githubusercontent.com/61879508/80504605-5333ba80-8941-11ea-9e50-22547ce1030c.png)

![image](https://user-images.githubusercontent.com/61879508/80504719-6e062f00-8941-11ea-9c20-7e3a606c2edc.png)

![image](https://user-images.githubusercontent.com/61879508/80504813-8b3afd80-8941-11ea-9389-89a61a17b3ff.png)

![image](https://user-images.githubusercontent.com/61879508/80504932-aefe4380-8941-11ea-9cbf-c4c650c7a52e.png)

![image](https://user-images.githubusercontent.com/61879508/84173026-b473a100-aa4a-11ea-92fc-90fe05af7c80.png)


</details>
</details>

**Comment by hamburglar2160** [comment]
<details>
<summary>
**Youtube TV**
</summary>
<details>
<summary>
Android Chromecast Controller:
</summary>

![image](https://user-images.githubusercontent.com/61879508/80501943-0d292780-893e-11ea-9006-3ab29fa3576e.png)
![image](https://user-images.githubusercontent.com/61879508/80501967-12867200-893e-11ea-9c4c-33f8fa14bd1d.png)
![image](https://user-images.githubusercontent.com/61879508/80501978-1619f900-893e-11ea-8eaa-5812e97c8f9e.png)
Stats screen
![image](https://user-images.githubusercontent.com/61879508/80502004-1d410700-893e-11ea-8951-e64da1234cda.png)
</details>
<details>
<summary>
Web
</summary>

![image](https://user-images.githubusercontent.com/61879508/80505411-49f71d80-8942-11ea-99c8-2dd6b2fbd2f5.png)

</details>
</details>

**Comment by hamburglar2160** [comment]
<details>
<summary>
Poweramp [Android music player]
</summary>
Player UI with user-selected background

![image](https://user-images.githubusercontent.com/61879508/80553143-c4518d00-8996-11ea-818c-517ca7804c2c.png)

Three dotted menu
![image](https://user-images.githubusercontent.com/61879508/80553157-cddaf500-8996-11ea-9ed2-6dd9ba9196a5.png)
Info/tags
![image](https://user-images.githubusercontent.com/61879508/80553163-d59a9980-8996-11ea-80b0-5743fbb994a0.png)
Chromecast controller
![image](https://user-images.githubusercontent.com/61879508/80553168-dcc1a780-8996-11ea-8c7c-04db096a1c4b.png)
Plater UI settings
![image](https://user-images.githubusercontent.com/61879508/80553176-e2b78880-8996-11ea-9a01-40d6a5fce64f.png)

</details>

**Comment by hamburglar2160** [comment]
<details>
<summary>
Alamo Drafthouse (web)
</summary>
Homescreen with 'welcome'

![image](https://user-images.githubusercontent.com/61879508/81353287-a53db400-9096-11ea-875f-3d96c90240b3.png)

Homescreen without 'welcome'
![image](https://user-images.githubusercontent.com/61879508/81353160-5c85fb00-9096-11ea-9837-63dcab5736fa.png)
Collections example
![image](https://user-images.githubusercontent.com/61879508/81353826-19c52280-9098-11ea-87cd-b9d8e4a67244.png)
Movie details
![image](https://user-images.githubusercontent.com/61879508/81353869-34979700-9098-11ea-8e12-348fdaa2c9e8.png)

</details>

**Comment by dougimus** [comment]
<details>
<summary>Spotify desktop</summary>

![2020-05-08](https://user-images.githubusercontent.com/30300064/81451490-a207fe00-9152-11ea-9920-bbc951564375.png)

</details>

**Comment by KucharczykL** [comment]
HBO GO

<details><summary>Desktop</summary>

Home
![Home](https://user-images.githubusercontent.com/31072879/82309476-7b07c280-99c3-11ea-8c21-fcd84a59190d.jpg)

Series Detail
![Series Detail](https://user-images.githubusercontent.com/31072879/82309540-8f4bbf80-99c3-11ea-961a-40dbb7ae560e.png)

Episode Detail
![Episode Detail](https://user-images.githubusercontent.com/31072879/82309629-a4285300-99c3-11ea-8aa7-572f3e52ca7f.png)

Player
![Player](https://user-images.githubusercontent.com/31072879/82309759-d0dc6a80-99c3-11ea-91f7-162ffff0c555.jpg)</details>

<details><summary>Android</summary>

Home
![Home](https://user-images.githubusercontent.com/31072879/82309788-da65d280-99c3-11ea-84dc-2f2605f18046.jpg)

Series Detail
![Series Detail](https://user-images.githubusercontent.com/31072879/82309808-e2be0d80-99c3-11ea-8a82-6f5d4e17b4ea.jpg)

Episode Detail
![Episode Detail](https://user-images.githubusercontent.com/31072879/82309840-ece00c00-99c3-11ea-85e3-16a37357ea4a.jpg)</details>

**Comment by hamburglar2160** [comment]
<details>
<summary>
Deezer (web)
</summary>
Artist Details

![image](https://user-images.githubusercontent.com/61879508/87032001-eeeb6d80-c1b1-11ea-9a77-b06e8fc00987.png)


Album Details [The button to the left of the "+" on the minibar is a full-screen overlay of lyrics]

![image](https://user-images.githubusercontent.com/61879508/87031589-53f29380-c1b1-11ea-8f15-222ae8b10ecb.png)

Queue Overlay (pulls up from the bottom)
![image](https://user-images.githubusercontent.com/61879508/87031798-987e2f00-c1b1-11ea-8c83-82ad697df5dd.png)

</details>

**Comment by dkanada** [comment]
@MrTimscampi the image placement on that Deezer queue layout looks a lot nicer than what we have at the moment. It also more closely matches our item details page and would look excellent if we started using the queue for shows or movie marathons since they could use the poster card.

**Comment by camc314** [comment]
<details>
  <summary>PlexAmp iPad Pro</summary>

Now playing screen (dark)
![2020-07-10-PlexampPic1](https://user-images.githubusercontent.com/18101008/87148604-9944bd00-c2a6-11ea-91a2-b1d25166398f.PNG)

Now playing screen (dark)
<img alt="2020-07-10-PlexampPic3" src="https://user-images.githubusercontent.com/18101008/87148510-6b5f7880-c2a6-11ea-9ede-9745c47cbdca.PNG">

Home page
<img alt="2020-07-10-PlexampPic6" src="https://user-images.githubusercontent.com/18101008/87149218-c04fbe80-c2a7-11ea-8b5a-f2b136add147.PNG">

Now playing screen (dark)
<img alt="2020-07-10-PlexampPic2" src="https://user-images.githubusercontent.com/18101008/87148633-ab266000-c2a6-11ea-93e6-d4b6400ce43d.PNG">

Now playing screen (light)
<img width="1366" alt="2020-07-10-plexamppic4(light)" src="https://user-images.githubusercontent.com/18101008/87149648-8e8b2780-c2a8-11ea-99a0-b0fbd6ac93fb.png">

Now playing (scroll down to next up)
<img width="1366" alt="2020-07-10-PlexampPic5" src="https://user-images.githubusercontent.com/18101008/87149455-2b999080-c2a8-11ea-8543-927521425bfc.png">






</details>

**Comment by hamburglar2160** [comment]
<details>
<summary>
Qobuz (web)
</summary>
Artist Details

![image](https://user-images.githubusercontent.com/61879508/87152397-a0a69f00-c283-11ea-9aac-f5ef3b923d46.png)

Alternative with blurred art

![image](https://user-images.githubusercontent.com/61879508/87152534-e2cfe080-c283-11ea-8548-29c921327b73.png)

Album Details

![image](https://user-images.githubusercontent.com/61879508/87152619-0b57da80-c284-11ea-89ea-55d847aa085f.png)

Track Details (musicbrainz!)

![image](https://user-images.githubusercontent.com/61879508/87152704-2f1b2080-c284-11ea-96d3-4ca774232091.png)

Music Playback with Minibar and Queue

![image](https://user-images.githubusercontent.com/61879508/87152849-71dcf880-c284-11ea-86b4-1e76b5122975.png)

Fullscreen playback

![image](https://user-images.githubusercontent.com/61879508/87152930-989b2f00-c284-11ea-86fb-5f702fc30a82.png)

Playlist Blur/info

![image](https://user-images.githubusercontent.com/61879508/87153173-09dae200-c285-11ea-991c-4e909cf790cc.png)

![output](https://user-images.githubusercontent.com/61879508/87153549-abfaca00-c285-11ea-913b-4d40ffcb6c8e.gif)

</details>

**Comment by hamburglar2160** [comment]
<details>
<summary>
Tidal (web)
</summary>
Full Player

![image](https://user-images.githubusercontent.com/61879508/87154872-fa10cd00-c287-11ea-9bb6-21655e057d97.png)

Album Details

![image](https://user-images.githubusercontent.com/61879508/87154982-2593b780-c288-11ea-9fdd-65281b07804f.png)

Album Details with Metadata

![image](https://user-images.githubusercontent.com/61879508/87155167-79060580-c288-11ea-8782-99eff6aee524.png)


Artist Details

![image](https://user-images.githubusercontent.com/61879508/87155018-38a68780-c288-11ea-9db8-9ee72a9af2c5.png)

Searching for Kanye

![image](https://user-images.githubusercontent.com/61879508/87155088-55db5600-c288-11ea-90be-39f0aa95fadc.png)


</details>

**Comment by hamburglar2160** [comment]
<details>
<summary>
Primephonic (web) [Classical Music]
</summary>
Browse

![image](https://user-images.githubusercontent.com/61879508/87156164-3e04d180-c28a-11ea-9cd6-07f56862ef28.png)

Composer Details

![image](https://user-images.githubusercontent.com/61879508/87156238-5c6acd00-c28a-11ea-983f-dde597bced73.png)

Work Details

![image](https://user-images.githubusercontent.com/61879508/87156302-7ad0c880-c28a-11ea-9858-05e6820651ff.png)

</details>

**Comment by KucharczykL** [comment]
I know the UI is the same for all devices in PlexAmp but it looks so bad on iPad. So much wasted space.

**Comment by chander421** [comment]
<details>
<summary>Plex Artists Grid View</summary>

![plex artists grid view](https://user-images.githubusercontent.com/18112003/101246335-ec665300-3712-11eb-994f-a883ecd24fe8.jpg)

</details>

<details>
<summary>Plex Artists Summary View</summary>

![plex artists summary view](https://user-images.githubusercontent.com/18112003/101246794-c68e7d80-3715-11eb-91c1-e89bf7a85c24.jpg)

</details>

<details>
<summary>Plex Artist Page</summary>

![plex artist](https://user-images.githubusercontent.com/18112003/101246772-aeb6f980-3715-11eb-89f9-254b18fd01e6.jpg)

</details>

<details>
<summary>Plex Album Page</summary>

![plex album](https://user-images.githubusercontent.com/18112003/101246634-e70a0800-3714-11eb-817b-13fe13e4d761.jpg)

</details>

**Comment by chander421** [comment]
<details>
<summary>Roon Home</summary>

![roon home page](https://user-images.githubusercontent.com/18112003/101258365-533f3880-3722-11eb-892e-2dffdb5e53dc.jpg)

</details>


<details>
<summary>Roon Artists</summary>

![roon artists](https://user-images.githubusercontent.com/18112003/101258280-c3998a00-3721-11eb-9505-c78fb4944e2e.jpg)

</details>

<details>
<summary>Roon Artist</summary>

![roon artist page](https://user-images.githubusercontent.com/18112003/101258271-b41a4100-3721-11eb-8f8d-f5cb71c09a12.jpg)

</details>

<details>
<summary>Roon Album</summary>

![roon album page](https://user-images.githubusercontent.com/18112003/101258291-ddd36800-3721-11eb-8b34-2bf49e403b14.jpg)

</details>

<details>
<summary>Roon Genre</summary>

![roon genre page](https://user-images.githubusercontent.com/18112003/101258376-6225eb00-3722-11eb-9dee-0d4ddd77b50c.jpg)

</details>

<details>
<summary>Roon Discover</summary>

![roon discover page](https://user-images.githubusercontent.com/18112003/101258382-6eaa4380-3722-11eb-8c50-5f09f1a09bfc.jpg)

</details>

**Comment by hamburglar2160** [comment]
<details>
<summary>
Amazon Music Prime (web)
</summary>
Full Screen Player

![image](https://user-images.githubusercontent.com/61879508/106063531-56252c80-60c6-11eb-8961-320be568ac48.png)

Queue Management
![image](https://user-images.githubusercontent.com/61879508/106063643-84a30780-60c6-11eb-8f68-32ccfc431029.png)

Album Details
![image](https://user-images.githubusercontent.com/61879508/106063690-9f757c00-60c6-11eb-83cb-9b3fdd72b36f.png)

Artist Details
![image](https://user-images.githubusercontent.com/61879508/106064123-5245da00-60c7-11eb-81df-00841abe5245.png)

</details>

**Comment by heyhippari** [comment]
Roon

<details>
<summary>
Scanning status
</summary>

![image](https://user-images.githubusercontent.com/19396809/126070255-e929228f-0b13-46b8-8580-424d13268e58.png)

</details>

<details>
<summary>
Album editor
</summary>

![Screenshot from 2021-07-18 15-20-27](https://user-images.githubusercontent.com/19396809/126070281-67285e59-af57-4218-bc27-bc6e22f79d64.png)

![Screenshot from 2021-07-18 15-20-42](https://user-images.githubusercontent.com/19396809/126070289-7db8dd01-d67f-4777-9610-022f09ceb76b.png)

![Screenshot from 2021-07-18 15-20-55](https://user-images.githubusercontent.com/19396809/126070291-fdcffc43-cd7f-4659-b309-288a6d95f08d.png)

![Screenshot from 2021-07-18 15-21-26](https://user-images.githubusercontent.com/19396809/126070293-8b1da43b-8d1a-4388-b34c-673557e36e78.png)

![Screenshot from 2021-07-18 15-21-53](https://user-images.githubusercontent.com/19396809/126070294-8546aac8-d7ad-413d-988e-ab5ffbb11cf9.png)

![Screenshot from 2021-07-18 15-29-18](https://user-images.githubusercontent.com/19396809/126070451-52546f29-2cc2-4c7b-9dc4-ab4081a07bdb.png)

</details>

<details>
<summary>
Discover
</summary>

![Screenshot from 2021-07-18 15-25-13](https://user-images.githubusercontent.com/19396809/126070431-71f540c3-5e6b-4b00-b96c-c04fa211433d.png)

</details>

<details>
<summary>
Album details
</summary>

![Screenshot from 2021-07-18 15-26-19](https://user-images.githubusercontent.com/19396809/126070474-521a452c-675b-4717-a6dc-64cabe47992c.png)

![Screenshot from 2021-07-18 15-33-17](https://user-images.githubusercontent.com/19396809/126070476-b188d9b6-ebb5-42fa-a150-8802d7edb10a.png)

![Screenshot from 2021-07-18 15-33-23](https://user-images.githubusercontent.com/19396809/126070477-d619f9b9-51ec-4474-94f2-50e82730349c.png)

![Screenshot from 2021-07-18 15-34-16](https://user-images.githubusercontent.com/19396809/126070478-5e2c52ba-30fb-4862-9e31-3e04575f2f5e.png)

![Screenshot from 2021-07-18 15-34-55](https://user-images.githubusercontent.com/19396809/126070479-c214ae07-c9ab-4460-8487-4f02c284d38a.png)

![Screenshot from 2021-07-18 15-41-52](https://user-images.githubusercontent.com/19396809/126070528-fff69c66-3ffe-4f69-ab5a-8d837cf51c8e.png)

![Screenshot from 2021-07-18 15-47-31](https://user-images.githubusercontent.com/19396809/126070529-51b80554-7790-45c7-a0ec-28cef351c248.png)

</details>

<details>
<summary>
Fullscreen player
</summary>

![Screenshot from 2021-07-18 15-35-50](https://user-images.githubusercontent.com/19396809/126070538-621f325b-6b7b-408b-8417-8070fc8095f5.png)

</details>
